//
// Student License - for use by students to meet course requirements and
// perform academic research at degree granting institutions only.  Not
// for government, commercial, or other organizational use.
//
// abcd_O_P.h
//
// Code generation for function 'abcd_O_P'
//

#ifndef ABCD_O_P_H
#define ABCD_O_P_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void abcd_O_P(double P_init, double V_init, double A_init, double P_wayp,
              double V_max, double A_max, double J_max, double J_min,
              double t[7]);

#endif
// End of code generation (abcd_O_P.h)
